config.yaml

```
@@ -2211,6 +2211,10 @@ Images:
   - v1.38.1
   - v1.38.3
   TargetImageName: mirrored-calico-operator
+- SourceImage: quay.io/tigera/operator
+  Tags:
+  - v1.38.3
+  TargetImageName: quay.io/tigera/operator
```

```suggestion
  TargetImageName: mirrored-calico-operator
```
@adamkpickering 
I think we want all of these under the existing `TargetImageName`, maybe a bug in the image-mirror scripting?